### PR TITLE
additional destroy logic

### PR DESCRIPTION
**Pull request by zspitzer**
need to also unsubscribe from global events to prevent a memory leak #87

### CODE REVIEW

File: src/jSignature.js [hunk 1]
```diff
@@ -1356,6 +1356,9 @@ var GlobalJSignatureObjectInitializer = function(window){
 						sig.$controlbarLower.remove();
 						sig.$controlbarUpper.remove();
 						$(sig.canvas).remove();
+						var globalEvents = $.fn[apinamespace]('globalEvents');
+						for (var e in sig.eventTokens)
+							globalEvents.unsubscribe(sig.eventTokens[e]);
```
Comment (funkyOne) [inline src/jSignature.js]: Maybe couple of slight adjustments will improve merge probability:
- it's better to use braces with multiline blocks 
- should use hasOwnProperty in for..in

Comment (zspitzer) [inline src/jSignature.js]: thanks, changes applied

File: src/jSignature.js [hunk 2]
```diff
@@ -1357,8 +1357,11 @@ var GlobalJSignatureObjectInitializer = function(window){
 						sig.$controlbarUpper.remove();
 						$(sig.canvas).remove();
 						var globalEvents = $.fn[apinamespace]('globalEvents');
-						for (var e in sig.eventTokens)
-							globalEvents.unsubscribe(sig.eventTokens[e]);
+						for (var e in sig.eventTokens){
+							if (sig.eventTokens.hasOwnProperty(e){
```
Comment (brinley) [inline src/jSignature.js]: Its missing a closing bracket.

File: src/jSignature.js [hunk 3]
```diff
@@ -1357,8 +1357,11 @@ var GlobalJSignatureObjectInitializer = function(window){
 						sig.$controlbarUpper.remove();
 						$(sig.canvas).remove();
 						var globalEvents = $.fn[apinamespace]('globalEvents');
```
Comment (brinley) [inline src/jSignature.js]: Just realised that globalEvents should be available and this line might be redundant. Can you verify and remove?

Comment (zspitzer) [inline src/jSignature.js]: done

File: src/jSignature.js (hunk 2 above)
Comment (zspitzer) [inline src/jSignature.js]: also fixed

**Pull request merged**